cmd/node/config/config.toml

```diff
@@ -17,6 +17,9 @@
    # DisableBuiltInFunctions represents the flag to enable/disable built in functions calls
    DisableBuiltInFunctions = false
```

Comment:
Shouldn't this flag be set to false?

Reply:
for development it will stay false - in order to test it. for mainnet false for the first 30 days.

Reply:
for mainnet true, right?

Reply:
yes. true for all.

Reply:
ok